main/ui/src/main/java/org/cryptomator/ui/controllers/MainController.java

```diff
@@ -216,7 +216,27 @@ public void initStage(Stage stage) {
 
 	private void gracefulShutdown() {
 		vaults.filtered(Vault.NOT_LOCKED).forEach(Vault::prepareForShutdown);
-		Platform.runLater(Platform::exit);
+		if (!vaults.filtered(Vault.NOT_LOCKED).isEmpty()) {
+			ButtonType tryAgainButtonType = new ButtonType(
```

Comment:
can you configure line wrap after 220 chars? most IDEs default to 80, which feels like punch card era :D

Reply:
I changed it, is this better?